medical_insurance/models/medical_insurance_plan.py

```diff
@@ -30,3 +36,40 @@ class MedicalInsurancePlan(models.Model):
     member_exp = fields.Date(
         string='Expiration Date',
     )
+    notes = fields.Text(
+        string='Extra Info',
+        help='Additional Information',
+    )
+
+    #Plan Template Information
```

Comment:
This comment isn't necessary IMO. It's also failing lint because there should be a space between the `#` and the comment itself:
```
# My comment
```